### PR TITLE
feat: #434 Change the time count  for a week and two weeks

### PR DESCRIPTION
**Pull request by Alex89198900**
1.  Change the day count
2.  Refactor

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export const WayStatistic = (props: WayStatisticProps) => {
 
   const averageTimeForJob = Math.round(totalWayTime / allJobs.length);
 
-  const lastWeekDayReports = props.dayReports.filter((dayReport) => dayReport.createdAt > lastWeekDate);
+  const lastWeekDayReports = props.dayReports.filter((dayReport) => new Date(dayReport.createdAt.toDateString()) >= lastWeekDate);
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx:110]: `new Date(dayReport.createdAt.toDateString())` looks a little hard to read. And also look like it is repeatable logic? Should we put it in some DateUtils helper and reuse everywhere?

**Review by tritonJS826 — APPROVED** [review]

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]
Please rename PR and comment according to the docs)

File: mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx]: Done

File: mw-webapp/src/utils/DateUtils.ts [hunk 2]
```diff
@@ -23,7 +23,14 @@ export class DateUtils {
...
 
-    return new Date(dateInPast);
+    return new Date(dateInPast.toDateString());
+  }
+
+  /**
+   * Change date to date yyyy-mm-dd
+   */
+  public static getToDate(date: Date): Date {
+    return new Date (date.toDateString());
```
Comment (tritonJS826) [inline mw-webapp/src/utils/DateUtils.ts:33]: As I see the method returns not yyyy-mm-dd, but Date object. Don't be so fast. Take a few minutes to describe utils properly. They could be highly reusable

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/utils/DateUtils.ts (hunk 2 above)
Comment (tritonJS826) [reply mw-webapp/src/utils/DateUtils.ts:33]: Lets rename: getToDate -> roundToDate

Comment (Alex89198900) [reply mw-webapp/src/utils/DateUtils.ts]: Done

**Review by Ekaterina1994 — COMMENTED** [review]
PLease check your PR naming )))) PLease read carefully our docs)

**Review by SergioAJS — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**